commons/src/main/java/io/hyscale/commons/commands/provider/ImageCommandProvider.java

```diff
@@ -200,6 +201,12 @@ public String dockerImageByNameFilterByImageOwner(String imageName) {
         StringBuilder sb = new StringBuilder(dockerImages());
         return sb.append(SPACE).append(imageName).append(filter(IMAGE_OWNER, HYSCALE)).append(quiet()).toString();
     }
+    
+    public Map<String, String> getImageLabelMap(){
+        Map<String, String> labels = new HashMap<>();
+        labels.put(IMAGE_OWNER, HYSCALE);
+        return labels;
+    }
```

Comment:
Get this out in a different class, feels like it is not part of command but to an Image.
The contract has changed